### ISSUE TITLE
Bug with dependency. You cant use BuildBattle on servers 1.12 and lower

### ISSUE THREAD

**Issue by Freyhold** [Drawethree/PRO-BuildBattle#5]
**Describe the bug**
On servers - 1.12 or lower. U cant use buildbattle because it only can hook worldguard just 7.0+, but worldguard is only 1.13(version 7.0+). So that dependency make this plugin onlny 1.13 + 
**To Reproduce**
Steps to reproduce the behavior. How the bug appeared ?

**Expected behavior**
Works on all versions. 

**Server Information (please complete the following information):**
 - Java Version [e.g 8]
 - Minecraft Server Version 1.12.2

**Comment by Flashback083** [comment]
Yep, i have same issue, i hope the dev will can fix it cause in spigot page its says its work with 1.12 :/
